daemon/algod/api/server/v2/test/helpers.go

```diff
@@ -238,6 +240,21 @@ func (m *mockNode) AbortCatchup(catchpoint string) error {
 	return m.err
 }
 
+func (m *mockNode) SetBlockTimeStampOffset(offset int64) error {
+	if !m.devmode {
+		return fmt.Errorf("cannot set block timestamp when not in dev mode")
+	}
+	return nil
+}
+
+func (m *mockNode) GetBlockTimeStampOffset() *int64 {
```

Comment:
nit: I see some of the other functions in the file utilize the mocks' `m.err`. This might be a way to simulate certain error conditions in your test, if you think more coverage is appropriate.